query-coordinator/src/main/scala/com/socrata/querycoordinator/QueryRewriter.scala

```diff
@@ -820,13 +823,16 @@ object QueryRewriter {
   }
 
   /**
-    * Filter rollups to non-compound query, i.e. leaf
+    * Merge rollups analysis
     */
-  def simpleRollups(rus: Map[RollupName, BinaryTree[Anal]]): Map[RollupName, Anal] = {
-    rus.filter {
-      case (_, Leaf(_)) => true
-      case _ => false
-    }.mapValues(_.outputSchema.leaf)
+  def mergeRollupsAnalysis(rus: Map[RollupName, BinaryTree[Anal]]): Map[RollupName, Anal] = {
```

Comment:
Nit: what about `mergeRollupAnalyses` instead of `mergeRollupsAnalysis` to make this name a little more gramatical?